### ISSUE TITLE
Improve time zone support

### ISSUE THREAD

**Issue by jakehobbs** [dxe/alc-mobile-api#14]
The admin pages should show times in the user's local time zone instead of being hardcoded to show Pacific time.

**Comment by AlexisDeschamps** [comment]
@jakehobbs Do you still want to add this to the product? It should be possible by doing something like this: https://github.com/dxe/alc-mobile-api/pull/23

**Comment by jakehobbs** [comment]
I don't think this necessarily needs to get done for this year's ALC conference, but it would be good to do eventually if the app will be used more in the future, especially if it's used in a different region.

**Comment by jakehobbs** [comment]
I'm a little nervous to change much right now before ALC because everything is working and we don't have much automated testing. How do you feel about waiting to make this change after the conference?

I don't think it needs to happen before then, since everything happening during ALC 2021 is PDT.
